message/validation/errors.go

```diff
@@ -82,6 +82,7 @@ var (
 	ErrUnexpectedConsensusMessage              = Error{text: "unexpected consensus message for this role", reject: true}
 	ErrNoSigners                               = Error{text: "no signers", reject: true}
 	ErrWrongRSASignatureSize                   = Error{text: "wrong RSA signature size", reject: true}
+	ErrWrongBLSSignatureSize                   = Error{text: "wrong RSA signature size", reject: true}
```

Comment:
typo
```suggestion
	ErrWrongBLSSignatureSize                   = Error{text: "wrong BLS signature size", reject: true}
```

Reply:
New message validation has 100% test coverage, could you please add a test for this one?

But since it has ssz size assertion, I'm not sure if it's possible, so probably we'll need to drop this check and replace it with a comment

Reply:
Yup, replacing it with a comment. Thanks!